openpype/hosts/houdini/plugins/publish/validate_houdini_license_category.py

```diff
@@ -3,30 +3,29 @@
 from openpype.pipeline import PublishValidationError
 
 
-class ValidateHoudiniCommercialLicense(pyblish.api.InstancePlugin):
-    """Validate the Houdini instance runs a Commercial license.
+class ValidateHoudiniNotApprenticeLicense(pyblish.api.InstancePlugin):
+    """Validate the Houdini instance runs a non Apprentice license.
 
-    When extracting USD files from a non-commercial Houdini license, even with
-    Houdini Indie license, the resulting files will get "scrambled" with
-    a license protection and get a special .usdnc or .usdlc suffix.
+    When extracting USD files from an apprentice Houdini license,
+    the resulting files will get "scrambled" with a license protection
+    and get a special .usdnc or .usdlc suffix.
```

Comment:
```suggestion
    and get a special .usdnc suffix.
```

The `.usdlc` extension was from the previous Houdini Indie restrictions over three years ago (basically the name refers to: usd lesser commercial). That restriction has been lifted.